crates/phala-types/src/lib.rs

```diff
@@ -717,6 +717,7 @@ pub mod worker_endpoint_v1 {
     #[derive(Encode, Decode, Debug, Clone, PartialEq, Eq, TypeInfo)]
     pub struct EndpointInfo {
         pub endpoint: Vec<u8>,
+        pub block_time: u64,
```

Comment:
Good to add a block time, but we should also enforce the check on the blockchain.

Reply:
Done